site/docs/hive.md

```diff
@@ -17,113 +17,211 @@
+SET iceberg.catalog.glue.catalog-impl=org.apache.iceberg.aws.GlueCatalog;
+SET iceberg.catalog.glue.warehouse=s3://my-bucket/my/key/prefix;
+SET iceberg.catalog.glue.lock-impl=org.apache.iceberg.aws.glue.DynamoLockManager;
+SET iceberg.catalog.glue.lock.table=myGlueLockTable
+```
+
+## DDL Commands
+
+### CREATE EXTERNAL TABLE
+
+The `CREATE EXTERNAL TABLE` command is used to overlay a Hive table "on top of" an existing Iceberg table. 
+Iceberg tables are created using either a [`Catalog`](./javadoc/master/index.html?org/apache/iceberg/catalog/Catalog.html),
+or an implementation of the [`Tables`](./javadoc/master/index.html?org/apache/iceberg/Tables.html) interface,
+and Hive needs to be configured accordingly to operate on these different types of table.
+
+#### Path-based Hadoop tables
+
+Iceberg tables created using `HadoopTables` are stored entirely in a directory in a filesystem like HDFS.
+You can use other compute engines or the Java/Python API to create such a table. 
+
+Suppose there is a table `table_a` and the table location is `hdfs://some_path/table_a`. 
+Now overlay a Hive table on top of this Iceberg table by issuing Hive DDL like so:
+
 ```sql
-SELECT * from table_b;
+CREATE EXTERNAL TABLE table_a 
+STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler' 
+LOCATION 'hdfs://some_bucket/some_path/table_a';
 ```
 
-#### Using Hadoop Catalog
-Iceberg tables created using `HadoopCatalog` are stored entirely in a directory in a filesystem like HDFS.
+#### Hadoop catalog tables
+
+Iceberg tables created using `HadoopCatalog` are stored entirely in a directory in a filesystem like HDFS, 
+similar to the tables created through `HadoopTables`.
+You can use other compute engines or Java/Python API to create such as table. 
 
-##### Create an Iceberg table
-The first step is to create an Iceberg table using the Spark/Java/Python API and `HadoopCatalog`. For the purposes of this documentation we will assume that the fully qualified table identifier is `database_a.table_c` and that the Hadoop Catalog warehouse location is `hdfs://some_bucket/path_to_hadoop_warehouse`. Iceberg will therefore create the table at the location `hdfs://some_bucket/path_to_hadoop_warehouse/database_a/table_c`.
+Suppose there is a table `table_b` and the table location is `hdfs://some_path/table_b`. 
 
-##### Create a Hive table
 Now overlay a Hive table on top of this Iceberg table by issuing Hive DDL like so:
+
 ```sql
-CREATE EXTERNAL TABLE database_a.table_c 
+CREATE EXTERNAL TABLE database_a.table_b
 STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler' 
-LOCATION 'hdfs://some_bucket/path_to_hadoop_warehouse/database_a/table_c'
+LOCATION 'hdfs://some_path/table_b'
 TBLPROPERTIES (
   'iceberg.mr.catalog'='hadoop', 
-  'iceberg.mr.catalog.hadoop.warehouse.location'='hdfs://some_bucket/path_to_hadoop_warehouse')
-;
+  'iceberg.mr.catalog.hadoop.warehouse.location'='hdfs://some_bucket/path_to_hadoop_warehouse'
```

Comment:
I am a little bit confused.
I think we do not want to promote this type of catalog configuration anymore. We kept this for backward compatibility, but we would like to deprecate it later.

AFAIK at table creation the warehouse location does not need to be set. Only the catalog should be set, and the identifier.

Reply:
+1 for depracating `'iceberg.mr.catalog'`. We should use `iceberg.catalog=catalog_name`

Reply:
Sounds good, I was just unclear if we want to actually still have it in doc or not because it was in the original doc. I will completely remove the reference to `iceberg.mr.catalog` then.

Reply:
I also marked the related public static variables as deprecated.

Reply:
Thanks!